packages/core/helper-plugin/src/utils/getDisplayName/index.js

```diff
@@ -0,0 +1,22 @@
+const getDisplayName = ({ firstname, lastname, username, email }, formatMessage) => {
```

Comment:
Think this should have some JSDoc around it to explain it a lot more 👍🏼 

You could if you felt comfortable do the `@type` annotation